src/transformers/models/emformer/configuration_emformer.py

```diff
@@ -0,0 +1,154 @@
+            the `inputs_ids` passed when calling [`EmformerModel`]. Vocabulary size of the model. Defines the different
+            tokens that can be represented by the *inputs_ids* passed to the forward method of [`EmformerModel`].
+        num_hidden_layers (`int`, *optional*, defaults to 20):
+            Number of hidden layers in the Transformer encoder.
+        num_attention_heads (`int`, *optional*, defaults to 8):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (`int`, *optional*, defaults to 2048):
+            Dimensionality of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_act (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
+            `"relu"`, `"selu"` and `"gelu_new"` are supported.
+        hidden_dropout (`float`, *optional*, defaults to 0.1):
+            The dropout probability for all fully connected layers in the embeddings, encoder, and pooler.
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        blank_token_id (`int`, *optional*, defaults to 4096):
+            The ID of a blank token used in RNN-T decoding.
+        pad_token_id (`int`, *optional*, defaults to 1):
+            The ID of a padding token used to pad [`EmformerForRNNT`] outputs.
+        input_dim (`int`, *optional*, defaults to 80):
+            The feature dimension of input features (Mel spectrogram).
+        time_reduction_input_dim (`int`, *optional*, defaults to 128):
+            The dimension that the input features are projected to before time shuffling.
+        time_reduction_stride (`int`, *optional*, defaults to 4):
+            The stride of [`EmformerTimeReduction`] used to compress input features into fewer frames.
+        left_context_length (`int`, *optional*, defaults to 30):
+            The length of Emformer attention context on the left.
+        right_context_length (`int`, *optional*, defaults to 4):
+            The length of Emformer attention context on the right.
+        segment_length (`int`, *optional*, defaults to 16):
+            The length of Emformer attention segments.
+        output_dim (`int`, *optional*, defaults to 1024):
+            The dimension that the Transformer output features are projected to.
+        token_embedding_dim (`int`, *optional*, defaults to 512):
+            The dimensionality of RNN-T token embeddings.
+        num_lstm_layers (`int`, *optional*, defaults to 3):
+            The number of LSTM layers in the RNN-T predictor.
+
+    Example:
+
+    ```python
+    >>> from transformers import EmformerModel, EmformerConfig
+
+    >>> # Initializing a Emformer anton-l/emformer-base-librispeech style configuration
```

Comment:
same here -> let's move it to the facebook org (all checkpoint names below as well)